tests/main/test_homePage.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+@pytest.mark.django_db
+class TestHomepageView:
+    def test_user_redirected_to_admin_login(self, client):
```

Comment:
Doesn't this test check redirect from the admin page?
```suggestion
    def test_user_redirected_from_admin_page(self, client):
```

Reply:
Yes, I changed it.